lerna.json

```
@@ -1,10 +1,18 @@
 {
   "lerna": "2.5.1",
+  "version": "independent",
+  "npmClient": "npm",
   "packages": [
     "elements/*"
   ],
-  "version": "1.10.1",
+  "command": {
+    "publish": {
+      "message": "chore(release): publish version bumps"
+    }
+  },
   "ignoreChanges": [
-    "elements/*/dist/**/*"
+    "elements/*/{dist,_temp}/**/*",
+    "elements/*/{demo,docs,test}/**/*",
+    "elements/*/*"
```

```suggestion
    "elements/*/{demo,docs,test}/**/*",
    "elements/*/*.txt"
```

@mwcz pointed out that we'd need to roll a patch for README changes where we could have an error published to npm (no way to correct that without a release).  The gulpfile is also able to influence the rendered output so best to include that in versioning too